src/app/app.component.ts

```diff
@@ -167,6 +168,7 @@ export class AppComponent implements OnInit, AfterViewInit, CanvasTableSelectLis
     private hotkeysService: HotkeysService,
     public settingsService: AppSettingsService,
     private savedSearchService: SavedSearchesService,
+    _usage: UsageReportsService,
```

Comment:
this doesnt get set / used?

Reply:
It is not used anywhere, but putting it here makes Angular instantiate it the moment we open the webmail for the first time (but no sooner), so that we can start periodically checking webmail's settings in there. I couldn't think of any more elegant way to do it – any ideas?

Reply:
Umm, nope other than adding a comment for our future selves